src/App.jsx

```diff
@@ -14,70 +16,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGoods] = useState('Jam');
```

Comment:
the state stores one entity, and the name of the setter says that there is more than one entity in the state
so you should rename to `setGood`